tests/python_package_test/test_dask.py

```diff
@@ -409,7 +418,7 @@ def test_ranker(output, client, listen_port, group):
     # have high rank correlation with scores from serial ranker.
     dcor = spearmanr(rnkvec_dask, y).correlation
     assert dcor > 0.6
-    assert spearmanr(rnkvec_dask, rnkvec_local).correlation > 0.75
+    assert spearmanr(rnkvec_dask, rnkvec_local).correlation > 0.8
```

Comment:
nice! I'm comfortable with this, given the distributed you saw in https://github.com/microsoft/LightGBM/issues/3817#issuecomment-770426671